dotcom-rendering/src/components/CricketScorecard.tsx

```diff
@@ -0,0 +1,379 @@
+			[centre-column-end];
+	}
+`;
+
+const getExtrasDescription = ({
+	byes,
+	legByes,
+	wides,
+	noBalls,
+	penalties,
+}: Extras): ReactNode => (
+	// Return HTML because we might be able to give these more accessible descriptions
+	<>
+		{byes > 0 && `${byes}b `}
+		{legByes > 0 && `${legByes}lb `}
+		{wides > 0 && `${wides}w `}
+		{noBalls > 0 && `${noBalls}nb `}
+		{penalties > 0 && `${penalties}p`}
+	</>
+);
+
+type BowlerData = {
+	name: string;
+	overs: number;
+	maidens: number;
+	runs: number;
+	wickets: number;
+	balls: number;
+};
+
+type BatterData = {
+	name: string;
+	ballsFaced: number;
+	runs: number;
+	fours: number;
+	sixes: number;
+	howOut: string;
+};
+
+type Extras = {
+	byes: number;
+	legByes: number;
+	noBalls: number;
+	penalties: number;
+	wides: number;
+};
+
+type InningsTotals = {
+	runs: number;
+	overs: string;
+	wickets: number;
+	extras: number;
+};
+
+type FallOfWicketData = {
+	order: number;
+	name: string;
+	runs: number;
+};
+
+type CricketTeam = {
+	name: string;
+	lineup: string[];
+};
+
+type InningsData = {
+	description: string;
+	bowlers: BowlerData[];
+	batters: BatterData[];
+	extras: Extras;
+	inningsTotals: InningsTotals;
+	fallOfWickets: FallOfWicketData[];
+};
+
+const Bowling = ({ bowlers }: { bowlers: BowlerData[] }) => (
+	<table css={tableStyles}>
+		<thead>
+			<tr>
+				<th>Bowler</th>
+				<th>O</th>
+				<th>M</th>
+				<th>R</th>
+				<th>W</th>
+			</tr>
+		</thead>
+		<tbody>
+			{bowlers.map((bowler) => (
+				<tr key={bowler.name}>
+					<th scope="row">{bowler.name}</th>
+					<td>
+						{bowler.overs}.{bowler.balls % 6}
+					</td>
+					<td>{bowler.maidens}</td>
+					<td>{bowler.runs}</td>
+					<td>{bowler.wickets}</td>
+				</tr>
+			))}
+		</tbody>
+	</table>
+);
+
+const Batting = ({
+	batters,
+	extras,
+	inningsTotals,
+}: {
+	batters: BatterData[];
+	extras: Extras;
+	inningsTotals: InningsTotals;
+}) => (
+	<table
+		css={[
+			tableStyles,
+			css`
+				${until.leftCol} {
+					border-top: 0.0625rem solid
+						${palette('--football-match-list-border')};
+				}
+			`,
+		]}
+	>
+		<thead>
+			<tr>
+				<th>Batter</th>
+				<th css={hideUntilTabletStyle}></th>
+				<th>Runs</th>
+				<th>Balls</th>
+				<th css={hideUntilTabletStyle}>4s</th>
+				<th css={hideUntilTabletStyle}>6s</th>
+			</tr>
+		</thead>
+		<tbody>
+			{batters.map((batter) => (
+				<tr key={batter.name}>
+					<th scope="row">
+						<strong>{batter.name}</strong>
+						<div css={hideFromTabletStyle}>{batter.howOut}</div>
+					</th>
+					<td css={hideUntilTabletStyle}>{batter.howOut}</td>
+					<td>{batter.runs}</td>
+					<td>{batter.ballsFaced}</td>
+					<td css={hideUntilTabletStyle}>{batter.fours}</td>
+					<td css={hideUntilTabletStyle}>{batter.sixes}</td>
+				</tr>
+			))}
+			<tr
+				css={css`
+					th,
+					td {
+						border-top: 0.0625rem dashed
+							${palette('--cricket-scorecard-divider')};
+					}
+				`}
+			>
+				<th scope="row">
+					<strong>Extras</strong>
```

Comment:
What's the purpose of the `strong` tag here?

Reply:
I think we thought it was "important" based on the docs [^1] but re-reading the examples it seems wrong, will remove and just apply the style

[^1]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong